pkg/api/norman/customization/project/project_actions.go

```diff
@@ -450,3 +459,32 @@ func getID(id interface{}) (string, error) {
 func isProvisionedRke2Cluster(cluster *v3.Cluster) bool {
 	return cluster.Status.Provider == v32.ClusterDriverRke2 && imported.IsAdministratedByProvisioningCluster(cluster)
 }
+
+func areK3SPodSecurityPoliciesEnabled(clusterv3 *v3.Cluster, clusterv1 *v1.Cluster) bool {
+	if clusterv3.Status.Provider != v32.ClusterDriverK3s {
+		return false
+	}
+	args := parseKubeAPIServerArgs(clusterv1)
+	return strings.Contains(args["enable-admission-plugins"], "PodSecurityPolicy")
+}
+
+// parseKubeApiServerArgs parses the "kube-apiserver-arg" available in the
+// clusters' MachineGlobalConfig to a map. The arguments are expected to
+// follow the "key=value" format. Arguments that don't follow this format
+// are ignored.
+func parseKubeAPIServerArgs(cluster *v1.Cluster) map[string]string {
+	result := map[string]string{}
+	args := cluster.Spec.RKEConfig.MachineGlobalConfig.Data["kube-apiserver-arg"].([]any)
+	for _, arg := range args {
+		key, value, found := strings.Cut(arg.(string), "=")
+		if found {
+			result[key] = value
+		} else {
+			log.Warnf(
```

Comment:
Are there other areas of the norman package that log? Just curious, haven't looked into it too much

Reply:
No there are no other instances. Maybe it's best to do something else

Reply:
I decided to just remove it and ignore invalid args if any.